src/Entities/Transformers/Family.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Choredo\Entities\Transformers;
```

Comment:
IMO Transformers shouldn't be in the Entities Directory. It's not a sub-concern of the entity itself. I'd prefer `Choredo\Transformers` What are your thoughts?